frontend/views/containers/global-dashboard/NewsAndUpdates.vue

```diff
@@ -97,6 +109,54 @@ export default ({
   color: $text_1;
 }
 
+.c-loader-skeleton-container {
+  position: relative;
+  display: block;
+
+  .c-skeleton-block {
+    display: block;
+
+    &:not(:last-of-type) {
+      margin-bottom: 4rem;
+    }
+
+    .c-loading-box {
+      animation: loading-heartbeat 3s linear infinite;
+      background-color: var(--general_1);
+      opacity: 0.625;
+    }
+
+    .c-skeleton-date {
+      margin-bottom: 0.625rem;
+      width: 8.75rem;
+      min-height: 0;
+      height: 1.25rem;
+      border-radius: $radius-large;
+    }
+
+    .c-skeleton-card {
+      width: 100%;
+      min-height: 0;
+      height: 10rem;
+      border-radius: $radius-large;
+    }
+  }
+}
+
+@keyframes loading-heartbeat {
+  0% { background-color: var(--general_1); }
+  50% { background-color: var(--general_0); }
+  100% { background-color: var(--general_1); }
```

Comment:
@SebinSong in most other places you use `$general_1` etc instead of `var(--general_1)`. Is this intentional?